src/main/features/core/windowTitleInfo.js

```diff
@@ -0,0 +1,16 @@
+const windowTitle = WindowManager.get(global.mainWindowID).getTitle();
+let songInfo = {};
+
+if (Settings.get('enableWindowTitleInfo')) {
+  PlaybackAPI.on('change:song', (_songInfo) => {
+    songInfo = _songInfo;
+    WindowManager.get(global.mainWindowID).setTitle(
+      (_songInfo.title && _songInfo.artist) ? `${songInfo.title} - ${songInfo.artist}` : windowTitle
```

Comment:
Are those songs you uploaded your self?

I'd much rather go for
```js
(songInfo.title || 'Unknown Song')
(songInfo.artist || 'Unknown Artist')
```

kind of thing